katas/square-digits/jeanpeare/squareDigits.js

```diff
@@ -1,3 +1,12 @@
 export function squareDigits(num) {
-    return 0;
+    let divideArr = [...`${num}`].map(c => parseInt(c)); // Divide input number in an array
```

Comment:
It is good to add inline comments from time to time, but in this case you might not need to include one comment for each line of your code. Instead, try to use more meaningful variable names so you need to add less inline comments